src/SFA.DAS.Roatp.CourseManagement.Web/Views/Standards/ConfirmDeleteStandard.cshtml

```diff
@@ -0,0 +1,117 @@
+﻿@using SFA.DAS.Roatp.CourseManagement.Web.Models.Standards
+@model ConfirmDeleteStandardViewModel
+
+@{
+    ViewBag.Title = "Confirm deletion";
+}
+@section breadcrumb {
+<a class="govuk-back-link" href="@Model.BackUrl">Back</a>
+}
+<main class="govuk-main-wrapper " id="main-content" role="main">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_validationSummary" /> 
+            <div class="govuk-caption-xl">Manage a standard</div>
+            <h1 class="govuk-heading-xl">Are you sure you want to delete this standard?</h1>
+            <form method="post" asp-route="@RouteNames.PostConfirmDeleteStandard" enctype="multipart/form-data" novalidate>
+                <input type="hidden" asp-for="BackUrl" />
+                <input type="hidden" asp-for="CancelUrl" />
+                <div class="grid-row">
+                    <div class="column-two-thirds">
+                        <table class="govuk-table" aria-describedby="ConfirmRegulatedStandard">
+                            <thead class="govuk-table__head">
+                                <tr class="govuk-table__row">
+                                    <th class="govuk-table__header" scope="col" colspan="2"> </th>
+                                </tr>
+                            </thead>
+                            <tbody class="govuk-table__body">
+                                <tr class="govuk-table__row">
+                                    <td class="govuk-table__cell">
+                                        <span class="app-task-list__task-name">
+                                            <strong>
+                                                Standard
+                                            </strong>
+                                        </span>
+                                    </td>
+                                    <td class="govuk-table__cell">
+                                        @Model.StandardInformation.CourseDisplayName
```

Comment:
Use the partial view for standard information.

Reply:
@raviudari01 an explanation of what is not same would have helped. Any how I think the standard information should look same no matter where it is.

Reply:
The layout of the partial is quite different from the layout required for the details on the 'confirm standard to delete' page.  I cannot attach screenshots here to demonstrate, but will attach them to the story, to illustrate this.